packages/generate-scaffold/README.md

```diff
@@ -0,0 +1,33 @@
+ # Generator-Webpack-Scaffold
+
+> This package will let you create your own custom webpack scaffold
+
+
+
+
+
+
+
+
+## Installation
+.
+
+```bash
+npm install
+npm run run
+```
+
+Then The generated scaffold:
```

Comment:
Imagine telling a story here. Write some text to explain what the installation does.

Reply:
Yeah sure.

Reply:
done